packages/wallets/src/features/cashier/modules/WithdrawalFiat/WithdrawalFiat.tsx

```diff
@@ -1,12 +1,17 @@
 import React, { useEffect } from 'react';
 import { useCashierFiatAddress } from '@deriv/api';
-import { Loader } from '../../../../components';
-import { WithdrawalVerificationModule } from '../WithdrawalVerification';
+import { Loader, WalletsErrorScreen } from '../../../../components';
+import { isServerError } from '../../../../utils/utils';
+
 import './WithdrawalFiat.scss';
 
 const WithdrawalFiat = () => {
     const verificationCode = sessionStorage.getItem('verification_code');
```

Comment:
I know it was already added in previous PRs, but is there any reason for using sessionStorage?
Even if its 'just' session storage, it still does not sound good to store verification code at all.
Verification codes of any type should be used on spot and then discarded.

Reply:
cc @markwylde-deriv

Reply:
Previously, we used to set `verification_code` in client store. Since wallets package doesn't have access to stores, we set and get it from session storage.

Reply:
I can not find `sessionStorage.setItem('verification_code')` anywhere in our code. Where/what sets this?

Reply:
if its actually local storage, then definitely cannot be there, as it means, that verification code gonna be persisted even after logging out or invalidating access tokens on server side...

Reply:
We're manually removing the `verification_code` from localStorage after its use, but yes; there are edge cases where the code can persist in localStorage and will be a valid code until expiry.
Could you check `redirect.jsx` in `core`? That's where `setVerificationCode` sets everything in localStorage right now, we might require a refactor there. Do you have any suggestion?

Reply:
Why does it need to be stored at all? Don't we send this verification_code to the server as soon as the user has typed/pasted it in?

Reply:
Hi guys, i have one suggestion. What if we put the verification code in query param when we redirecting from `redirect.jsx`. So, it looks like `history.push(routes.wallets_withdrawal)?verification_code={code}`. And then in component where we use the code, we remove it from url or route to `history.push(routes.wallets_withdrawal)` after we use it.